src/redis-benchmark.c

```diff
@@ -354,15 +354,14 @@ static redisConfig *getRedisConfig(const char *ip, int port,
     redisFree(c);
     return cfg;
 fail:
-    fprintf(stderr, "ERROR: failed to fetch CONFIG from ");
-    if (hostsocket == NULL) fprintf(stderr, "%s:%d\n", ip, port);
-    else fprintf(stderr, "%s\n", hostsocket);
-    int abort_test = 0;
     if (reply && reply->type == REDIS_REPLY_ERROR &&
-        (!strncmp(reply->str,"NOAUTH",6) ||
-         !strncmp(reply->str,"WRONGPASS",9) ||
-         !strncmp(reply->str,"NOPERM",6)))
+        !strncmp(reply->str,"NOAUTH",6)) {
```

Comment:
i think WRONGPASS should be fatal too.
not sure about NOPERM since arguably the user could be missing access to CONFIG but have access to the commands they want to benchmark.

Reply:
we would not get WRONGPASS error, since `getRedisContext` can catch it.